plugins/inputs/dpdk/dpdk.go

```diff
@@ -0,0 +1,240 @@
+// +build linux
+
+package dpdk
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/influxdata/telegraf/config"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	jsonparser "github.com/influxdata/telegraf/plugins/parsers/json"
+)
+
+const (
+	dpdkDescription  = "Reads metrics from DPDK applications using v2 telemetry interface."
+	dpdkSampleConfig = `
```

Comment:
Please name those `description` and `sampleConfig` to match most other plugins.

Reply:
Renamed.